src/preseed/rc.local.first

```diff
@@ -1,40 +1,7 @@
 #!/bin/sh
 
-# Kremlor Package Archive Keyring ( for OCEMR )
-
-dpkg -i /var/packages/repo/kremlor-archive-keyring_*.deb
-
-# Watch4inet ( to commect over cell-modem )
-
-dpkg -i /var/packages/repo/tcl8.4_*.deb /var/packages/repo/tcl_*.deb
-dpkg -i /var/packages/repo/usb-modeswitch_*.deb /var/packages/repo/usb-modeswitch-data_*.deb
-dpkg -i /var/packages/repo/watch4inet_*.deb
-
-# CFEngine
-
-dpkg -i /var/packages/repo/cfengine3_*.deb
-
-# Appliance packages
-
-dpkg -i /var/packages/repo/ocemr-appliance_*.deb
-dpkg -i /var/packages/repo/ocemr-appd_*.deb
-
-# Run cfengine...
-
-/usr/sbin/cf-agent -IKf /var/cfengine/masterfiles/failsafe.cf
-sleep 2
-/usr/sbin/cf-agent -IKf /var/cfengine/masterfiles/failsafe.cf
-sleep 2
-/usr/sbin/cf-agent -IK
-sleep 2
-/usr/sbin/cf-agent -IK
-sleep 2
-/usr/sbin/cf-agent -IK
+# Run ansible
+cd /root/ansible/
+ansible-playbook ./ocemr.yml
```

Comment:
`root@ocemr-server:~/ansible# ansible-playbook -vvvD ./ocemr.yml `:

```
TASK [ocemr : Install ocemr] ***********************************************************************************
task path: /root/ansible/ocemr/tasks/main.yml:12
fatal: [localhost]: FAILED! => {
    "msg": "The conditional check 'app_server' failed. The error was: error while evaluating conditional (app_server): 'app_server' is undefined\n\nThe error appears to have been in '/root/ansible/ocemr/tasks/main.yml': line 12, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Install ocemr\n  ^ here\n"
}
        to retry, use: --limit @/root/ansible/ocemr.retry

PLAY RECAP *****************************************************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=1   
```

Reply:
Should work with the added yml.